packages/odyssey-react/src/components/Form/Form.module.scss

```diff
@@ -26,7 +26,7 @@
 .header,
 .error,
 .main {
-  margin-block-end: var(--SpaceMarginBlockEndHeaderErrorMain);
+  margin-block-end: var(--HeaderErrorMainMarginBlockEnd);
```

Comment:
Wordy, but adheres to the schema.

We could split these up, but I believe the design intent is for them to be uniform. I think we need a better ruling on when Design Intent trumps Customization.

Reply:
Thanks for calling this out, I had the same thought!